### PR TITLE
Totul e un vis

### PR DESCRIPTION
**Pull request by solidgriever**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2200)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: romanian/somn/inapoi_la_somn.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Se pare ca inca visez vise ciudate.
+
+Da... doar un vis ciudat... Daca adorm din nou sunt sigur ca o sa ma trezesc la realitate...
+
+HA HA HA ha haaa, totul doar in mintea mea. Noapte Buna!!
+
+[Te trezesti la realitate](sfarsit/the_end.md)
```
Comment (casey-collab) [inline romanian/somn/inapoi_la_somn.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: romanian/sfarsit/the_end.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Dupa ce te trezesti, simti de parca ai pierdut o oportunitate uriasa.
```
Comment (casey-collab) [inline romanian/sfarsit/the_end.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: romanian/sfarsit/the_end.md (hunk 2 above)
Comment (cbuckey-uda) [inline romanian/sfarsit/the_end.md]: If you move the directory `sfarsit` and its files inside of the `somn` directory, that should fix the link, and I think it's also a better organization.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**